src/mon/PGMap.cc

```diff
@@ -4550,8 +4556,41 @@ int reweight::by_utilization(
       osd_util.second = (double)p.second.kb_used / (double)p.second.kb;
     }
     util_by_osd.push_back(osd_util);
+
+    // Overweighted devices and their weights are factors to
```

Comment:
@tchaikov I guess, @amitkumar50 is correct.
[This](https://en.wikibooks.org/wiki/C%2B%2B_Programming/Code/Style_Conventions/Comments) (the note at the bottom) explains, why we should shy away from using single line comments.